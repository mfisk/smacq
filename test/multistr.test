#!/bin/sh

${BINDIR}smacqq -m -O <<EOF 
print cmd from tabularinput(-f, $TEST/multistr.test, cmd, arg) where substr(cmd, "print")
print cmd from tabularinput(-d, " ", -f, $TEST/multistr.test, cmd, arg) where substr(cmd, "bin")
EOF
