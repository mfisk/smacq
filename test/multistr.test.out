print cmd from tabularinput(-f, multistr.test, cmd, arg) where substr(cmd, "print")
print cmd from tabularinput(-f, multistr.test, cmd, arg) where substr(cmd, "print")
print cmd from tabularinput(-d, " ", -f, multistr.test, cmd, arg) where substr(cmd, "bin")
